### PR TITLE
первый exam HTML_CSS

### PR DESCRIPTION
**Pull request by BiruliaAl**
k;

### CODE REVIEW

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 1]
```diff
@@ -0,0 +1,646 @@
...
+    font-family: 'Open Sans';
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 32px;
+    color: #525C60;
+}
+.catalog {
+    background-color: #EFF6F1;
+}
+.catalog__wrapper {
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css:512]: gap: 30px

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-family: 'Roboto', sans-serif;
+    font-family: 'Yellowtail', cursive;
+    letter-spacing: 0.36px;
+}
+
+.container {
+    padding: 0 20px;
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css]: padding: 0 20px; - delete this for dekstop

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 3]
```diff
@@ -0,0 +1,646 @@
...
+
+.customer hr {
+    margin: 80px 0;
+}
+.customer__statistics {
+    display: flex;
+    justify-content: space-between;
+}
+
+.customer__statistic__item {
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css:412]: circles should be round how in design

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 4]
```diff
@@ -0,0 +1,646 @@
...
+    font-family: 'Roboto';
+    font-weight: 700;
+    font-size: 25px;
+    text-align: center;
+    color: #274C5B;
+    width: auto;
+}
+.letter__background {
+    background: url(../images/img/footer_img.png) center center/cover no-repeat;
+    min-height: 320px;
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css]: 120px

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 5]
```diff
@@ -0,0 +1,646 @@
...
+.letter__form .button__item {
+    background-color: #335B6B;
+}
+footer {
+    padding: 100px 0;
+}
+
+.footer {
+    display: flex;
+    justify-content: space-around;
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css:588]: padding-bottom: 85px;

File: BiruliaAlena/HTML_CSS/css/style_media.css [hunk 6]
```diff
@@ -0,0 +1,379 @@
+/* 2560, 1920,
+1440, 1280, 1152, 1024, 800, 768, 640, 425, 375, 320. */
+
+@media (max-width: 1350px) {
+    .header__nav {
+        justify-content: space-evenly;
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style_media.css:11]: .container {
    max-width: 1100px;
    margin: 0 auto;
} use this change style max-width at different media-request

File: BiruliaAlena/HTML_CSS/css/style.css [hunk 7]
```diff
@@ -0,0 +1,646 @@
...
+    background-color: #EFD372;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    border: none;
+}
+.welcome__descr button p,
+.offer__descr button p {
+    color: #274C5B;
+}
+
+.promo {
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style.css:171]: 90px 0 140px 0

File: BiruliaAlena/HTML_CSS/css/style_media.css [hunk 8]
```diff
@@ -0,0 +1,379 @@
...
+    .footer__bot__big {
+        display: none;
+    }
+    .footer__bot__small {
+        display: block;
+    }
+}
+
+@media (max-width: 900px) {
+    .header__nav {
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style_media.css:83]: very big header is 217px - fix

File: BiruliaAlena/HTML_CSS/css/style_media.css [hunk 9]
```diff
@@ -0,0 +1,379 @@
...
+        margin-top: 20px;
+    }
+    .header__button__item {
+        padding: 8px;
+    }
+    .welcome__wrapper {
+        min-height: 710px;
+    }
+    .promo,
+    .about,
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style_media.css:101]: where correct pattern for .about display grid

File: BiruliaAlena/HTML_CSS/css/style_media.css [hunk 10]
```diff
@@ -0,0 +1,379 @@
...
+   .catalog__wrapper {
+    padding: 0;
+    flex-direction: column;
+   }
+
+    .customer {
+        padding: 10px 10px;
+    }    
+}
+@media (max-width: 390px) {
```
Comment (kino1ol) [inline BiruliaAlena/HTML_CSS/css/style_media.css:367]: .advantage {
background-image: url(./images/img/advantages_img.png);
    background-size: cover;
}
.advantage__wrapper {
min-height: 594px;
}
.advantage_descr {
top: 50%;
    transform: translateY(-50%);
}
use this please ^-)

**Review by kino1ol — APPROVED** [review]
please fix my comment and review again all media-request, good job)

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 1 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css:512]: +

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 2 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css]: +

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 3 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css:412]: how to implement it correctly?

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 4 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css]: corrected by 120. but in the layout 320

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 5 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css:588]: why?

File: BiruliaAlena/HTML_CSS/css/style_media.css (hunk 6 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style_media.css:11]: +

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 7 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css:171]: +

File: BiruliaAlena/HTML_CSS/css/style_media.css (hunk 9 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style_media.css:101]: +

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 3 above)
Comment (kino1ol) [reply BiruliaAlena/HTML_CSS/css/style.css:412]: set width in px and padding could be 50px 50px

Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css:412]: +

File: BiruliaAlena/HTML_CSS/css/style_media.css (hunk 10 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style_media.css:367]: +

File: BiruliaAlena/HTML_CSS/css/style_media.css (hunk 8 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style_media.css:83]: +

File: BiruliaAlena/HTML_CSS/css/style.css (hunk 4 above)
Comment (BiruliaAl) [reply BiruliaAlena/HTML_CSS/css/style.css]: +

**Comment by kino1ol** [comment]
good

**Pull request merged**